virttest/nfs.py

```diff
@@ -336,8 +336,9 @@ def cleanup(self, ssh_auto_recover=True):
             raise exceptions.TestFail("Failed to umount %s" % self.mount_dir)
 
         # Recover SSH connection
-        self.ssh_obj.auto_recover = ssh_auto_recover
-        del self.ssh_obj
+        if ssh_auto_recover:
+            self.ssh_obj.auto_recover = ssh_auto_recover
+            del self.ssh_obj
```

Comment:
Looks like this needs a relook as `self.ssh_obj` is getting deleted why update it?

Reply:
based on the update `__del__()` will work after `del self.ssh_obj` also it is suggested to set auto_recover as False by default which is taken care as well